tools/isolib/isolib.xml

```diff
@@ -0,0 +1,72 @@
+<tool id="isolib" name="isolib" version="1.0.0+galaxy0" profile="21.09">
+    <description>create an isotopic pattern library for given compounds and adducts</description>
+    <creator>
+        <person
+            givenName="Helge"
+            familyName="Hecht"
+            url="https://github.com/hechth"
+            identifier="0000-0001-6744-996X" />
+        <organization
+            url="https://www.recetox.muni.cz/"
+            email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+            name="RECETOX MUNI" />
+    </creator>
+    <edam_operations>
+        <edam_operation>operation_3632</edam_operation>
+    </edam_operations>
+    <requirements>
+        <requirement type="package" version="1.10.0">bioconductor-metabocoreutils</requirement>
+        <requirement type="package" version="1.12.0">bioconductor-spectra</requirement>
+        <requirement type="package" version="1.6.0">bioconductor-msbackendmsp</requirement>
+        <requirement type="package" version="2.6">r-envipat</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript -e 'source("${__tool_directory__}/isolib.R")' '${input_file}' '${adducts}' '${threshold}' '${append_adduct}' '${isotope_library}'
```

Comment:
I don't understand why source() is needed, I think you can remove that.

Reply:
true